### DISCUSSION TITLE
CSS in Div

### DISCUSSION THREAD

**Discussion by afoysal** [vercel/next.js#29573] [Help]
I am learning Next.js. I would like to add CSS in Div using Material UI. 

How can I do that ?

Thanks

**Comment by sanjaiyan-dev** [comment]
### Following example using 'styled' utility 

I am extremely sorry if I am wrong because I also started to learn Next Js recently and still learning it 😄 

```js
import {styled} from "@mui/material/styles"

const StyledDiv = styled('div')({
    backgroundColor: '#000028',
    borderRadius: 12,
    minHeight: '12vh'
})

function ComponentDiv(){
    return(
        <StyledDiv>
          Div
        </StyledDiv>
    )
}
```
